ui-ngx/src/app/modules/home/components/widget/lib/digital-gauge.ts

```diff
@@ -406,7 +406,7 @@ export class TbCanvasDigitalGauge {
             filter.transform(timestamp, this.localSettings.timestampFormat);
         }
         const value = tvPair[1];
-        if (value !== this.gauge.value) {
+        if (value != this.gauge.value) {
```

Comment:
It is bad practice to use automatic data type casting. The problem is that the check for the data type the widget receives is not used.

Reply:
changed to parseFloat of the value